DMTN-049.tex

```diff
@@ -123,14 +136,21 @@ \subsection{Federating a User-Generated Photo-$z$ Catalog}\label{ssec:time_ops_u
 
 \subsection{International Partnerships (In-Kind Contributions)}\label{ssec:time_inkind}
 
-All in-kind program contributions related to photo-$z$ will be subject to the processes outlined in this roadmap and will be included and integrated into the communal efforts to optimize the LSST photo-$z$.
-These contributions might include, e.g., individuals' expertise; scientific analyses; software algorithms, tools, or infrastructure; computational resources; datasets or observing time.
-These contributions might be directed to the Project (e.g., DM, commissioning team), and/or to the science community (e.g., a Science Collaboration, all data rights holders). 
+As the process for International Partnerships with Rubin Observatory via in-kind contributions evolved, photometric redshifts emerged as an area in which (1) multiple teams propose to provide different types of contributions and (2) optimizing the LSST scientific returns on these contributions requires the coordination across the Rubin community: the in-kind teams, Rubin staff, the Science Collaborations, and the broad community of data rights holders.
 
-In-kind teams should expect to work closely with the project (e.g., DM) and other Rubin community members on each step of this roadmap, including the solicitation of input from the science community, the process of implementation and validation, and maintaining and evolving their contributions during Rubin Operations (as applicable).
-In-kind contributors should also review the summary of related data products and documentation that might pertain to their contribution (Appendix \ref{sec:dp}), and the example scientific use-cases that the photo-$z$ estimates are expected to support (Appendix \ref{sec:use}).
+Thus, the stages of this roadmap and the Photo-$z$ Coordination Group (\S~\ref{ssec:time_pzcoord}) were developed with the in-kind contributions in mind, to create a timeline and communication methods that will assist with the development and integration of the in-kind program contributions related to photo-$z$.
+
+At the time of Version 4 of this document, the proposed contributions included, e.g., individuals' expertise; scientific analyses; software algorithms, tools, or infrastructure; computational resources; datasets or observing time.
+The proposals specify the recipients for these contributions as either the Rubin Observatory project (e.g., DM, commissioning) or the science community (e.g., a Science Collaboration, all data rights holders).
+
+Here are a couple of examples to illustrate how in-kind teams would be involved in this roadmap.
+Participation level would vary between in-kind teams depending on their type of contribution.
```

Comment:
"The participation level would ...." or "The level of participation would ..."

Reply:
fixed